docs/01-readme.md

```diff
@@ -113,10 +114,12 @@ Additionally, the following optional dependencies exist:
 ## Running Awesome
 
 You can directly select Awesome from your display manager. If not, you can
-add the following line to your `.xinitrc` to start Awesome using `startx`
+add the following line to the end of your `.xinitrc` to start Awesome using `startx`
 or to `.xsession` to start Awesome using your display manager:
 
     exec awesome
+    
+make sure this .xinit file is in your home directory. Do this by typing `vim ~/.xinitrc` and add the line above
 
 In order to connect Awesome to a specific display, make sure that
```

Comment:
Not directly related to your commit, but how does adding `exec awesome` to `~/.xinitrc` connects the WM to a specific display ?

I'm not an Xorg expert but since `xrandr` works regardless of the WM/DE being used. Maybe this line should change as well in this patch